src/main/java/Event.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * The Event class represents a type of task.
+ */
+public class Event extends Task {
+
+    protected String when;
```

Comment:
When could be renamed to a more semantic 'period', or a seperate 'start', and 'end date'!